crates/sui-types/src/sui_system_state/epoch_start_sui_system_state.rs

```diff
@@ -44,14 +44,21 @@ impl EpochStartSystemState {
         }
     }
 
-    pub fn get_sui_committee(&self) -> Committee {
```

Comment:
Instead of changing this function, could you add a new function that return Vec<SeedPeer>?

Reply:
this function is not currently used